dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceFile.java

```diff
@@ -151,11 +152,15 @@ public void setPath(String path) {
     }
 
     public String getAbsolutePath() {
-        return absolutePath;
+        if (absolutePath == null) {
+            return null;
+        }
+        return Paths.get(absolutePath).normalize().toString();
```

Comment:
I don't fully understand the issue yet, but would it make sense to do this in `setAbsolutePath` instead? Or is the concern that there are already database entries where this needs to be corrected and it can't be addressed via migration?

Reply:
The latter-ish. 
The postgres migration creates non-normalized paths, so we need to correct on the way out.